plugins/modules/inventory_intent.py

```diff
@@ -263,6 +256,25 @@
         description: Location of the sites allocated for the APs
         type: list
         elements: str
+      provision_wired_device:
+        description: A list of dictionaries containing the IP address of wired devices and the site name where they will be provisioned.
+        type: list
+        elements: dict
+        suboptions:
+          device_ip:
+            description: The IP address of the wired device.
+            type: str
+          site_name:
+            description: The complete name of the site where the wired device will be provisioned(For example, "Global/USA/San Francisco/BGL_18/floor_pnp").
+            type: str
+          resync_retry_count:
+            description: The total number of retries to check whether the device has come to a managed state for provisioning.
+            type: int
+            default: 200
+          resync_retry_interval:
+            description: The interval (in seconds) at which the system will check the device status during provisioning.
```

Comment:
description: Sets the interval, in seconds, at which the system will recheck the device status throughout the provisioning process. If unspecified, the system will check the device status every 2 seconds by default.